share/jupyterhub/templates/login.html

```diff
@@ -10,6 +10,7 @@
 
 {% block login %}
 <div id="login-main" class="container">
+  <h1 class="sr-only">Login page</h1>
```

Comment:
I _think_ this h1 sould replace the current `div.auth-form-header` and visible, rather than hidden. Then we can make the `auth-form-header` style applied to the h1 instead of the div, but the same element.

Reply:
> I _think_ this h1 sould replace the current `div.auth-form-header` and visible, rather than hidden. Then we can make the `auth-form-header` style applied to the h1 instead of the div, but the same element.

Okay. That's better. I will make changes